package.json

```diff
@@ -41,7 +41,7 @@
     "isstream": "~0.1.1"
   },
   "scripts": {
-    "test": "npm run lint && node node_modules/.bin/taper tests/test-*.js && npm run test-browser && npm run clean",
+    "test": "npm run lint && node node_modules/.bin/taper tests/test-*.js && npm run test-browser",
```

Comment:
Ah, it seems like this is safe to remove then. npm won't look anywhere else for scripts to run (see [run](https://docs.npmjs.com/cli/run-script)).